ddtrace/settings/config.py

```diff
@@ -544,5 +623,35 @@ def _get_service(self, default=None):
 
     def __repr__(self):
         cls = self.__class__
-        integrations = ", ".join(self._config.keys())
+        integrations = ", ".join(self._integration_config.keys())
         return "{}.{}({})".format(cls.__module__, cls.__name__, integrations)
+
+    def _subscribe(self, items, handler):
+        # type: (List[str], Callable[[Config, List[str]], None]) -> None
+        self._subscriptions.append((items, handler))
```

Comment:
Might be going crazy but I don't see this function being used anywhere... should we automatically subscribe all configuration options added via `__setattr__()` or is that a future step?

Reply:
Yeah it's not being used yet in this PR. We will use it for future settings like `service` and `env`